packages/salesforcedx-aura-language-server/tsconfig.json

```diff
@@ -0,0 +1,50 @@
+{
+    "compilerOptions": {
```

Comment:
only override what's necessary but doesn't match the common top-level one.  recommend 
  "extends": "../../../tsconfig.common.json",
like all the other ones do